interface/main/messages/js/reminder_appts.js

```diff
@@ -0,0 +1,524 @@
+                                                                       // $("#msg_notes_"+r_uid).val(present+" "+material+" printed.\n");
+                                                                       //$("#msg_notes_"+r_uid).focus();
+                                                                       });
+                   
+                   //refresh_me();
+                   });
+        //
+    
+}
+
+
+$.date = function(dateObject) {
+    var d = new Date(dateObject);
+    var day = d.getDate();
+    var month = d.getMonth() + 1;
+    var year = d.getFullYear();
+    if (day < 10) {
+        day = "0" + day;
+    }
+    if (month < 10) {
+        month = "0" + month;
+    }
+    var date = day + "/" + month + "/" + year;
+    
+    return date;
+};
+
+$(function(){
+  /*
+   * this swallows backspace keys.
+   * stops backspace -> back a page in the browser, a very annoying thing indeed.
+   */
+  var rx = /INPUT|SELECT|TEXTAREA|SPAN|DIV/i;
+  
+  $(document).bind("keydown keypress", function(e){
+                   if( e.which == 8 ){ // 8 == backspace
+                   if(!rx.test(e.target.tagName) || e.target.disabled || e.target.readOnly ){
+                   e.preventDefault();
+                   }
+                   }
+                   });
+  });
+
+    // Open the add-event dialog.
+function newEvt(pid,pc_eid) {
+    var f = document.forms[0];
+    var url = '../../main/calendar/add_edit_event.php?patientid='+pid+'&eid='+pc_eid;
+        //    if (f.ProviderID && f.ProviderID.value) {url += '&userid=' + parseInt(f.ProviderID.value);}
+    dlgopen(url, '_blank', 800, 480);
+    return false;
+}
+
+function delete_Recall(pid,r_ID) {
+    if (confirm('Are you sure you want to delete this Recall?')) {
+            //top.restoreSession();
+    var url = 'save.php';
+    $.ajax({
+           type   	: 'POST',
+           url    	: url,
+           data     : {
+                'action': 'delete_Recall',
+                'pid' 	: pid,
+                'r_ID'  : r_ID
+           }
+           
+           }).done(function(result) {
+                   refresh_me();
+                   });
+    }
+
+    }
+function refresh_me() {
+    location.reload();
+}
+/**
+ *  This function is called when the user clicks the checkbox to "phone" the patient
+ *  Once checked, the checkbox is replaced with the current date and the Notes:input=text field is
+ *  populated with the current $username: <- and cursor is focused here.
+ *  When the Notes field is exited, function save_recall is fired to:
+ *		store the events in the DB
+ *		visually mark task 'completed' by changing its color designation to "Successfully Completed"
+ */
+function setPhone(r_uid,pc_eid) {
+    process_this('phone',r_uid,pc_eid);
+    var dateval = $.datepicker.formatDate('mm/dd/yy', new Date());
+    $("#msg_phone_"+r_uid).html(dateval);
+    $("#msg_notes_"+r_uid).focus();
+    var note = "<?php echo $_SESSION['authUser']; ?>: ";
```

Comment:
this will not work since this file will not go through the php compiler. this is the same reason why can't do translations in js scripts. If need this variable, recommend including this as a parameter in the setPhone function.